internal/node/controller/controller2.go

```diff
@@ -28,6 +32,7 @@ func NewController2(
 	pkgManagers map[node.DriverType]package_manager.PackageManager,
 	db hdb.Client,
 	proxyServer *reverse_proxy.ProxyServer,
+	pdsHost string,
```

Comment:
should we take in an xrpc client here instead? I'm thinking about if we want to start using the embedded pds methods we should use something more abstract

Reply:
oh hmmm. I was thinking we will have to construct multiple xrpc.Client's in here so we just want the pdsHost that is "tied" to this habitat node, but I guess it's fine if that's the xrpc.Client "tied" to the habitat node? unsure